test/test-crosstab.js

```diff
@@ -280,4 +280,76 @@ describe('crosstab', function () {
             });
         });
     });
+
+    describe('setup during load', function() {
+        var iframe;
+
+        beforeEach(function (done) {
+            window.done = done;
+            iframe = runIframe(function () {
+                crosstab(function () {
+                    addText('crosstab setup complete');
+                });
+                window.parent.done();
+            });
+        });
+
+        it('should stop on beforeunload event and recover when canceled', function (done) {
+            window.done = done;
+
+            iframe.run(function () {
+                crosstab(function() {
+
+                    window.parent.expect(crosstab.stopKeepalive).to.be.undefined;
+                    
+                    // FIXME use `new Event()` once PhantomJS 2.0 is available on npm
```

Comment:
these `FIXME`s are due to https://github.com/ariya/phantomjs/issues/11289
Once those are fixed we can use the not deprecated version of creating events.